mutiny-core/src/keymanager.rs

```diff
@@ -228,12 +228,9 @@ mod tests {
             .unwrap();
         let db = MutinyStorage::new("".to_string()).await.unwrap();
 
-        let wallet = Arc::new(MutinyWallet::new(
-            &mnemonic,
-            db,
-            Network::Testnet,
-            Arc::new(esplora),
-        ));
+        let wallet = Arc::new(
+            MutinyWallet::new(&mnemonic, db, Network::Testnet, Arc::new(esplora)).unwrap(),
```

Comment:
unwrap?

Reply:
this is in the test, we can't `?` because it returns `()` not a `Result`

Reply:
Oh didn't see it was in test